server/punishments.js

```diff
@@ -469,20 +480,19 @@ const Punishments = new (class {
 	 * @param {string} note
 	 */
 	appendSharedIp(ip, note) {
-		let buf = `${ip}\tSHARED\t${note}\r\n`;
```

Comment:
This is useful futureproofing.

Reply:
You mean if `sharedips.tsv` has more types in the future? I guess that makes.

Reply:
Reverted,